pkgs/top-level/static.nix

```diff
@@ -151,4 +148,7 @@ in {
     };
   };
 
+  unbound = super.unbound.override {
+    static = true;
```

Comment:
Maybe `enableStatic` for consistency?

Reply:
Oh, there does not seem to be a standard on naming things like this.

Reply:
Yeah it’s inconsistent… enableStatic usually implies that you should build static libraries. In this case the static is needed for building the binaries statically. That’s the distinction i’ve arrived at between the two.

Now that i think about it, these aren’t really needed for this pr so i could leave them out.